packages/webcomponents/src/api/services/__mocks__/payment.service.ts

```diff
@@ -0,0 +1,13 @@
+import paymentDetail from '../../mockData/mockPaymentDetailSuccess.json';
```

Comment:
I found out recently that if we create a mock inside a folder named `__mocks__` we can use this mock on a test file like this: (this folder should be in the same level of the module we want to mock)

`jest.mock('../../services/payment.service.ts`)`

just adding this before the other imports in the test file. 👍